docs/scripts/sphinx_build_link_check.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# halt script on error
+set -e
+# Build locally, and then check links
+sphinx-build -E -W -b linkcheck source build
```

Comment:
is there more sphinx can check here besides `linkcheck`?

Reply:
@majurg I activated Travis-CI for my fork of aboutcode and yes the build passed. Check my build [here](https://travis-ci.org/AyanSinhaMahapatra/aboutcode/builds). Also, sphinx-build checks for syntactical errors, and other common errors (the makefile also does the same), I double checked myself again locally by inserting common/complex errors and it works.

Reply:
I'm working on more tests and what can be done (linting and setting standards), this is just some basic tests.

Reply:
Ok should I wait to merge this then or do you wish to add those changes later?

Reply:
No, you need not wait to merge this PR. I'll add those in a separate PR later.